### ISSUE TITLE
CE-292 Confusing tab navigation on Explore page for screen readers

### ISSUE THREAD

**Issue by ChrisC** [LLK/scratch-www#7198]
### Expected Behavior

The tabs for filtering categories on the [Explore page](https://scratch.mit.edu/explore/projects/all) should have a simple description of the tab name along with a description of the interactive state (for example: "Projects tab selected" or "Studios tab" if not selected).

### Actual Behavior

Tabs are currently read by screen readers in a confusing way. Some tabs also include graphics without alternative text description. For example, a selected tab may be read as "Link Graphic tabs/projects-active", making it difficult for screen reader users to interpret what happens when a tab link is clicked. 

### System Details
Browser: Chrome/Firefox/Safari, Screen Reader: Jaws/NVDA/Voiceover

**Screenshots**
<img width="979" alt="image" src="https://user-images.githubusercontent.com/58723/201800248-a025c498-ead6-47d3-aecc-585a05eef59f.png">
